selfdrive/controls/lib/pathplanner.py

```diff
@@ -60,7 +60,7 @@ def update(self, sm, CP, VM):
     active = sm['controlsState'].active
 
     angle_offset_average = sm['liveParameters'].angleOffsetAverage
-    angle_offset_bias = sm['controlsState'].angleModelBias + angle_offset_average
```

Comment:
The angleModelBias value is hard-coded to 0 in controlsd, so there's no value in this parameter